### PR TITLE
GRW-1759 / fix maxLength of car register

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

fixed mac length for the car register input field. 

**Before**
Screenshot 2022-11-14 at 15 00 55

**After**
Screenshot 2022-11-14 at 15 00 42

## Justify why they are needed

previously you could add as many chars as you'd like. 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-1759]: https://hedvig.atlassian.net/browse/GRW-1759?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/store/src/components/PriceForm/CarRegistrationField.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const CarRegistrationNumberField = ({ field }: RegistrationFieldProps) =>
       label={field.label ? translateLabel(field.label) : undefined}
       pattern={CAR_REGISTRATION_NUMBER_REGEX}
       placeholder="ABC 123"
-      minLength={7}
+      maxLength={7}
```
Comment (gustaveen) [inline apps/store/src/components/PriceForm/CarRegistrationField.tsx:33]: Is `minLength` covered by the pattern instead?

**Review by gustaveen — APPROVED** [review]
🙌

File: apps/store/src/components/PriceForm/CarRegistrationField.tsx (hunk 1 above)
Comment (KarnellSchultz) [reply apps/store/src/components/PriceForm/CarRegistrationField.tsx:33]: It'll throw an error if you try to submit something that does not match the pattern or longer than the pattern. But the `maxLength` prevents the user from entering more than the expected number of chars.

**Comment by abderrahmane-tj** [comment]
@KarnellSchultz 👋🏼 is this repo meant to be open to the public? (and a couple more 👇🏼)
image

**Pull request merged**